content/v2.0/write-data/use-telegraf/auto-config/_index.md

```diff
@@ -42,6 +42,11 @@ for using Telegraf with InfluxDB v2.0._
    Plugins listed with a <span style="color:#32B08C">{{< icon "check" >}}</span>
    require no additional configuration.
    To configure a plugin or access plugin documentation, click the plugin name.
+
+   {{% note %}}
+   Not all available plugins are listed on this screen. For more information on manually configuring additional plugins, see [Manually add Telegraf plugins](/v2.0/write-data/use-telegraf/manual-config/).
```

Comment:
@sanderson , wasn't sure if I should use this link or the list of plugins link here. Both seemed clunky? What do you think?

Reply:
I think this is fine.